### ISSUE TITLE
Synchronous parsing error

### ISSUE THREAD

**Issue by dherault** [RubenVerborgh/N3.js#78]
Hi there,

Some of us live in a sync world. :bug: 

```javascript
const createRdfParser = require('n3').Parser;
const parser = createRdfParser();
console.log(parser.parse(ttlString));
```

Throws:
```
TypeError: this._prefixCallback is not a function
    at Object._readPrefixIRI (/home/david/dev/cool_project/node_modules/n3/lib/N3Parser.js:564:10)
    at /home/david/dev/cool_project/node_modules/n3/lib/N3Parser.js:852:42
    at Array.every (native)
    at Object.parse (/home/david/dev/cool_project/node_modules/n3/lib/N3Parser.js:851:35)
```

Here's a fix:
```javascript
const createRdfParser = require('n3').Parser;
const parser = createRdfParser();
parser._prefixCallback = () => null;
console.log(parser.parse(data));
```

Best,

**Comment by dherault** [comment]
Merci!

**Comment by RubenVerborgh** [comment]
Thanks for reporting; released in v0.8.2!
